### ISSUE TITLE
initial home view with a random gene URL

### ISSUE THREAD

**Issue by IgorRodchenkov** [PathwayCommons/pcviz#25]
For consistency (and who knows what else; e.g., this might be the reason for #24 as well), the initial pcviz view (using, e.g., BRCA1, picked by chance from the hard-coded list of 4 genes) should probably display (and use/route) the same URL in the browser address bar as if a user typed that gene symbol manually. I.e., currently, it shows up and stays if you refresh the page as:

```
http://www.pathwaycommons.org/pcviz/
```

, whereas it should be:

```
http://www.pathwaycommons.org/pcviz/#neighborhood/BRCA1
```

Attn: @armish , @onursumer

**Comment by armish** [comment]
This was a design choice I made in which I assumed whoever is landing on the PCViz without any query genes doesn't know what gene to search for at first. So I thought it might be a good idea to show a random network up there and in case they refresh the page, let them see new networks for example.

If we set the URL to whatever we are showing randomly on the homepage, it is basically locking the user in, which I don't like that much. Any particular reason to switch behaviour?
